packages/api/src/controllers/room.ts

```diff
@@ -110,10 +113,18 @@ app.post(
   authorizer({}),
   validatePost("room-egress-payload"),
   async (req, res) => {
-    const room = await getRoom(req);
+    await getRoom(req);
```

Comment:
I assume this is only called to validate the room exists? Could you add a short comment to make that clear?